### PR TITLE
Minor grammar and typo fixes in web-config docs

### PR DESCRIPTION
**Pull request by neomantra**

### CODE REVIEW

File: docs/web-configuration.md [hunk 1]
```diff
@@ -86,8 +86,8 @@ which will look something like:
 `$2y$10$X0h1gDsPszWURQaxFh.zoubFi6DXncSjhoQNJgRrnGs7EsimhC7zG`
 
 The cost (10 in the example) influences the time it takes for computing the
-hash. A higher cost will en up slowing down the authentication process.
-Depending on the machine, a cost of 10 will take about ~70ms where a cost of
+hash. A higher cost will end up slowing down the authentication process.
+Depending on the machine, a cost of 10 will take about ~70ms, whereas a cost of
 18 can take up to a few seconds. That hash will be computed on every
```
Comment (roidelapluie) [inline docs/web-configuration.md:91]:
```suggestion
18 can take up to a few seconds. That hash will be computed on the first authenticated HTTP request and then cached.
```

**Review by roidelapluie — COMMENTED** [review]
Thanks, while you are at it, would you mind fixing a small error we have?

Thanks!!

File: docs/web-configuration.md (hunk 1 above)
Comment (neomantra) [reply docs/web-configuration.md:91]: Sure, I just force-pushed with that change.

Comment (roidelapluie) [reply docs/web-configuration.md:91]: I do not see the commit

Comment (neomantra) [reply docs/web-configuration.md]: Problem was between keyboard and chair ;) should be there now!

**Review by roidelapluie — APPROVED** [review]
Thanks!

**Pull request merged**